mitodl/bootcamp-ecommerce
# Moved drawer rendering to top-level and removed temp page

#### What are the relevant tickets?
Refines #573, supports various dashboard UIs

#### What's this PR do?
Moves the drawer component rendering to the top level of the app so we don't render it in multiple places, and gets rid of the temporary drawer page

#### How should this be manually tested?
Apply this patch, visit the dashboard page, expand the application detail for some application, then click "View/Edit Profile". The drawer should open
```
diff --git a/static/js/pages/applications/ApplicationDashboardPage.js b/static/js/pages/applications/ApplicationDashboardPage.js
index bb648a2..652b003 100644
--- a/static/js/pages/applications/ApplicationDashboardPage.js
+++ b/static/js/pages/applications/ApplicationDashboardPage.js
@@ -67,7 +67,7 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
   }
 
   renderApplicationDetail = (applicationId: number) => {
-    const { allApplicationDetail } = this.props
+    const { allApplicationDetail, setDrawerOpen, setDrawerState } = this.props
 
     if (!allApplicationDetail || !allApplicationDetail[String(applicationId)]) {
       return null
@@ -76,7 +76,10 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
       <div className="row application-detail">
         <div className="col-12">
           <h3>Profile Information</h3>
-          <a className="btn-link">View/Edit Profile</a>
+          <a className="btn-link" onClick={() => {
+            setDrawerState("profileEdit")
+            setDrawerOpen(true)
+          }}>View/Edit Profile</a>
         </div>
         <div className="col-12">
           <h3>Resume or LinkedIn Profile</h3>
```
